docs/conf.py

```diff
@@ -151,7 +152,7 @@
 # The theme to use for HTML and HTML Help pages.  See the documentation for
 # a list of builtin themes.
 #
-html_theme = "sphinx_symbiflow_theme"
+#html_theme = "sphinx_symbiflow_theme"
```

Comment:
This changes the HTML theme, which should be unrelated to this PR. See:

* https://skywater-pdk.readthedocs.io/en/main/
* https://skywater-pdk--398.org.readthedocs.build/en/398/

Reply:
Sorry, had no intention to change that **theme** line.